python/python/lance/dataset.py

```diff
@@ -635,6 +678,23 @@ def join(
         """
         raise NotImplementedError("Versioning not yet supported in Rust")
 
+    def open_blobs(self, blobs: pa.Array):
+        """
+        Open blobs in the dataset
+
+        Parameters
+        ----------
+        blobs : pa.Array
+            An array of blob IDs to open
```

Comment:
Potential user questions: What are blob ids? Where do I get them?

Reply:
I cleaned this up a little and added a follow-up task for adding tutorial / example content for blobs